hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -457,13 +458,18 @@
       tests: ["i2c_target_bad_addr"]
     }
     {
-      name: interrupt_transaction
+      name: target_mode_glitch
       desc: '''
-            Create tests to interrupt a normal transaction by a new 'start' or 'reset'.
-            Run those tests with dut host and target mode if possible.
+            Test handling of RStart ot Stop glitches in Target mode
+
+           Stimulus:
+              - Configure DUT/Agent to Target/Host mode respectively
+              - Issue a new request(RStart) to DUT during an active transfer
+              - Stop current request(Stop) to DUT during an active transfer
+              - Continue issuing requests to check if DUT FSM handles the transaction correctly
 
             Checking:
-            Dut can accept a new transaction after discarding interrupted transaction.
+            Ensure all transactions including glitches are observed in ACQDAT FIFO
```

Comment:
```suggestion
            Ensure all transactions including glitches are observed in i2c.ACQDATA FIFO
```

Reply:
Updated as per comments in [d7ad31e](https://github.com/lowRISC/opentitan/compare/fd90c0095a271793d33af41ee3cca9481298aa41..d7ad31ebedff7e0f905a8b6298d6909417eb1cff)